api/debug.go

```diff
@@ -218,18 +217,18 @@ func (d *DebugAPI) TraceCall(
 			}
 			// Override account balance.
 			if account.Balance != nil {
-				opts = append(opts, query.WithStateOverrideBalance(addr, (*big.Int)(*account.Balance)))
+				opts = append(opts, query.WithStateOverrideBalance(addr, account.Balance.ToInt()))
```

Comment:
The changes in this file come from: https://github.com/ethereum/go-ethereum/commit/ac0f220040bd4271a040170bc0b039eb88dee9bd#diff-c426ecd2f7d247753b9ea8c1cc003f21fa412661c1f967d203d4edf8163da344